RxMediaPicker.xcodeproj/project.pbxproj

```diff
@@ -133,6 +134,23 @@
 		};
 /* End PBXResourcesBuildPhase section */
 
+/* Begin PBXShellScriptBuildPhase section */
```

Comment:
Whoops! That's totally a mistake on my end. The script should only be part of the Example target. I'll fix that up real quick.